.github/workflows/ci.yml

```diff
@@ -65,15 +104,19 @@ jobs:
 
     - name: Rcat UDP Test
       run: cd ${{github.workspace}} && ./examples/example_scripts/example_udp.sh
-      
+      timeout-minutes: 1
```

Comment:
Maybe we should do the `timeout-minutes` stuff for all test scripts (also the "without extensions" runs below)?